Factories/templates/CMakeLists.txt.tpl

```diff
@@ -9,9 +9,9 @@ include(CMSSW)
 
 # Ensure C++11 is available
```

Comment:
Actually we could set the `O3` flag here? Might not change a lot since we are limited by I/O anyway, but who knows...